v3.0/reference/tools/tidb-lightning/deployment.md

```diff
@@ -380,6 +388,8 @@ TiDB Lightning 可随 TiDB 集群一起用 [Ansible 部署](/how-to/deploy/orche
     pd-addr = "172.16.31.4:2379"
     # tidb-lightning 引用了 TiDB 库，而它自己会产生一些日志。此设置控制 TiDB 库的日志等级。
     log-level = "error"
+    # MySQL SQL Mode 配置
+    #sql-mode = ""
```

Comment:
```suggestion
    # sql-mode = ""
```

Reply:
I followed previous comment style of L271, L282, etc, which doesn't have a seperating space on commented config lines.

Reply:
That's not the correct format (very likely done by mistake). Please also change them and make sure that no such format is in the document, thanks!